lib/solidus_gdpr/data_segments/base.rb

```diff
@@ -40,7 +45,8 @@ def export
       # Erases all data contained in this segment.
       #
       # This is used by the {DataEraser} service.
-      def erase
+      # @param erased_email [String] it will replace the current one
```

Comment:
The description of the parameter should be phrased like "the email that will replace the current one".

Reply:
Also, you need a blank line before `@param` definitions.